src/fontra/client/web-components/menu-bar.js

```diff
@@ -1,14 +1,24 @@
 import * as html from "../core/html-utils.js";
 import { SimpleElement } from "../core/html-utils.js";
 import { MenuPanel } from "./menu-panel.js";
+import { themeColorCSS } from "./theme-support.js";
+
+const colors = {
+  "menu-top-bar-background-color": ["#eee", "#444"],
+  "menu-top-bar-border-color": ["#d3d3d3", "#2e2e2e"],
+  "menu-top-bar-link-hover": ["#e1e1e1", "rgb(47, 47, 47)"],
+};
 
 export class MenuBar extends SimpleElement {
   static styles = `
+
+  ${themeColorCSS(colors)}
+
   .menu-bar {
     display: flex;
     align-items: center;
     font-size: 1rem;
-    height:100%;
+    height: var(--menu-top-bar-height);
```

Comment:
The menu bar should not depend on an external var. Please keep it "100%" as it was, and if that doesn't work anymore, a different solution is needed.

Reply:
This is again a matter of composition: the *top bar* height will be specified with a var. The menu bar that it contains has no business with a hard coded height. This should (become) the hierarchy:

```

+-- top bar ---------------------------------------------------------------+
| +-------------------------------+        +--------------+  +-----------+ |
| | menu bar                      |        | project name |  | user icon | |
| +-------------------------------+        +--------------+  +-----------+ |
+--------------------------------------------------------------------------+
```

Reply:
Thanks. I changed it.